src/WholeBodyStateDisplay.cpp

```diff
@@ -27,14 +27,19 @@ WholeBodyStateDisplay::WholeBodyStateDisplay()
     : has_new_msg_(false),
       initialized_model_(false),
       force_threshold_(0.),
+      torque_threshold_(0.),
+      use_contact_status_in_zmp_(true),
       use_contact_status_in_cop_(true),
       use_contact_status_in_grf_(true),
       use_contact_status_in_support_(true),
       use_contact_status_in_friction_cone_(true),
+      grf_locate_at_cop_(true),
+      friction_cone_locate_at_cop_(true),
```

Comment:
I am not convinced if this should be by default.
But I am fine with it if you really believe is much easier to see these quantities at the CoP point.

Reply:
Yes, fair enough. Done